src/param/mfcc.py

```diff
@@ -138,3 +138,10 @@ def get_frequency(mel):
     """
 
     return 700 * (np.power(10, mel / 2595) - 1)
+
+
+def MfccLogarithm (MelFrame):
```

Comment:
To nie zadziała. "MelFrame" to nazwa klasy, więc nie możesz tak nazwać zmiennej. Poza tym w pythonie nazwy funkcji powinny być z małej litery i rozdzielane "_" (tutaj wystarczy logarithm)